examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -70,17 +70,17 @@ S_bcs = HorizontallyPeriodicBCs(top = BoundaryCondition(Flux, Qˢ))
 
 # ## Model instantiation
 #
-# We instantiate a horizontally-periodic `Model` on the CPU with on a `RegularCartesianGrid`, 
-# using a `FPlane` model for rotation (constant rotation rate), a linear equation 
-# of state for temperature and salinity, the Anisotropic Minimum Dissipation closure 
+# We instantiate a horizontally-periodic `Model` on the CPU with on a `RegularCartesianGrid`,
+# using a `FPlane` model for rotation (constant rotation rate), a linear equation
+# of state for temperature and salinity, the Anisotropic Minimum Dissipation closure
 # to model the effects of unresolved turbulence, and the previously defined boundary
 # conditions for `u`, `T`, and `S`. We also pass the evaporation rate to the container
 # model.parameters for use in the boundary condition function that calculates the salinity
 # flux.
 
 model = Model(
          architecture = CPU(),
-                 grid = RegularCartesianGrid(N=(Nz, Nz, Nz), L=(Δz*Nz, Δz*Nz, Δz*Nz)),
+                 grid = RegularCartesianGrid(size=(Nz, Nz, Nz), grid=(Δz*Nz, Δz*Nz, Δz*Nz)),
```

Comment:
should be `length` rather than `grid`? Does this run?

Reply:
Yes, thanks for catching that. Sloppy PR on my part.